### DISCUSSION TITLE
Error while updating a record [Apr 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1778] [Discord Threads]
> 🙋‍♂️ **EmilyF**  
#### I know that at one point I uploaded a plant description with a set of parentheses in it, but I've since deleted that entry, as well as every notes column in every table, and I still get this error message every time I try to add information to any of the columns.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/962108180631015485/Screen_Shot_2022-04-08_at_5.49.35_PM.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### does it happen on any particular cell update or always ?  
#  
> 🙋‍♂️ **EmilyF**  
#### (testing that now)  
#  
> 🙋‍♂️ **EmilyF**  
#### so far it looks like I'm only getting that particular error when I'm updating the "growth habit" column, but some of the other columns are just deleting the updates when I navigate to another table  
#  
> 🙋‍♂️ **EmilyF**  
#### the "growth habit" column is just a simple multiselect  
#  
> 💁 **o1lab**  
#### ok  
#  
> 💁 **o1lab**  
#### could you give a refresh and try again   
#### I wonder it has gotten to a some err state  
#  
> 💁 **o1lab**  
#### also if you an lookup network error logs that would be really helpful  
#  
> 🙋‍♂️ **EmilyF**  
#### I've done that before, but I don't know how to off the top of my head, do you have a link to a simple how to? I'm on a mac running 10.13  
#  
> 🙋‍♂️ **vijayrathore8492**  
#### <@700711994621165648> Is your Project metadata in sync?  
#  
> 🙋‍♂️ **EmilyF**  
#### I'm not sure  
#  
> 🙋‍♂️ **EmilyF**  
#### I could enable another account for now to check?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/962108181729919006)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
